builder.go

```diff
@@ -94,10 +95,30 @@ func buildAndRunDir(dir string, filters []string, goBuildTags string) error {
 	return nil
 }
 
+func goMod() string {
+	if _, err := os.Stat("go.mod"); os.IsNotExist(err) {
+		return ""
+	}
+	file, err := os.Open("go.mod")
+	defer file.Close()
+	if err != nil {
+		panic(err)
+	}
+	line, err := bufio.NewReader(file).ReadString('\n')
+	if err != nil {
+		panic(err)
+	}
+	return string(line[7 : len(line)-1])
+}
+
 // ToSlash is being used to coerce the different
 // os PathSeparators into the forward slash
 // as the forward slash is required by Go's import statement
 func assembleImportPath(file string) string {
+	module := goMod()
```

Comment:
nit but `if mod := goMod(); mod != "" {` is a little cleaner in this simple case.

Reply:
done